imports/plugins/included/product-variant/containers/gridItemControlsContainer.js

```diff
@@ -1,47 +1,50 @@
-import React, { Component } from "react";
+import React, { PureComponent } from "react";
```

Comment:
Also, if we are going to remove `container` from some file names because they are in a `container` directory we should probably do that everywhere.